src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties (see an exampl
+ * in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+
+  sourceString.trim().split(';')
+    .map(item => item.trim().split(':'))
```

Comment:
Good work, but you need to make a few corrections

You don't need trim() here, and to avoid using trim() in the future, you can add a map() after split() in which you will iterate over the values and add trim() to them

```
.map(item => item
        .split(':')
        .map(x => x.trim()))
```